sys-libs/libxcrypt/libxcrypt-4.4.28-r1.ebuild

```diff
@@ -165,6 +165,8 @@ get_xcpkgconfigdir() {
 
 multilib_src_configure() {
 	local -a myconf=(
+		--build=${CBUILD}
+		--host=${CTARGET}
```

Comment:
econf passes `--host=${CHOST}`, which is correct most of the time.

Instead of doing this, you should set CHOST=${CTARGET} at the top of the ebuild, as part of the CATEGORY logic.

Reply:
Hmm, I guess we are doing something similar in glibc. So never mind on this.